pkg/printers/internalversion/describe.go

```diff
@@ -2048,9 +2048,10 @@ func describeNode(node *api.Node, nodeNonTerminatedPodsList *api.PodList, events
 					c.Message)
 			}
 		}
-		addresses := make([]string, 0, len(node.Status.Addresses))
+
+		w.Write(LEVEL_0, "Addresses:\n")
 		for _, address := range node.Status.Addresses {
-			addresses = append(addresses, address.Address)
+			w.Write(LEVEL_0, " %s:\t%s\n", address.Type, address.Address)
```

Comment:
Use `LEVEL_1` which will indent it automatically, and remove your own space at the beginning of the string.